framework/pom.xml

```diff
@@ -147,6 +147,20 @@
 			<version>0.9.30</version>
 			<scope>test</scope>
 		</dependency>
+
+
+		<!-- https://mvnrepository.com/artifact/org.postgresql/postgresql -->
+		<dependency>
```

Comment:
Pokud vím, tak neřeší. Bude potřeba je tam přidat, ale dělala bych to až bude hotový config, ať to můžeme rovnou testovat.